modules/models.py

```diff
@@ -116,7 +117,7 @@ def load_model(model_name):
             print(f"Could not find {pt_model}, exiting...")
             exit()
 
-        model = load_quant(path_to_model, Path(f"models/{pt_model}"), 4)
+        model = load_quant(path_to_model, Path(f"{pt_path}"), bits)
```

Comment:
Ahh sorry about this -- looks like this issue was introduced by my changes in #210 . Thank you for catching that!

Would you be okay with changing this to:
```python
model = load_quant(path_to_model, pt_path, bits)
```
Since `pt_path` is already a `Path`?

Reply:
Yeah sure. I just left it as is to be safe